### PR TITLE
Oldvaccination

### PR DESCRIPTION
**Pull request by Kalyani790**

### CODE REVIEW

File: app/components/HeaderNotiIcon.tsx [hunk 1]
```diff
@@ -184,8 +184,19 @@ const HeaderNotiIcon = (props: any): any => {
               })
             }
             if (currentChildNotis.vcnotis) {
+              // 
               currentChildNotis.vcnotis.forEach((item: any) => {
+                console.log(item,"..currentChildNotis");
+                if(item.title=="vcNoti1"){
+                const vcNotisExists= getVaccinesForPeriodCount(allVaccineData, item.growth_period);
+                console.log(vcNotisExists,"..vcNotisExists..")
+                if(vcNotisExists!="" && vcNotisExists!=null && vcNotisExists!=undefined){
```
Comment (UnicefMinsk) [inline app/components/HeaderNotiIcon.tsx:193]: null and undefined represent the absence of a value, while an empty value typically refers to an empty string ('') or an empty array ([]) so it is recommended to check for null or undefined first before checking for an empty value.

File: app/components/HeaderNotiIcon.tsx [hunk 2]
```diff
@@ -184,8 +184,19 @@ const HeaderNotiIcon = (props: any): any => {
               })
             }
             if (currentChildNotis.vcnotis) {
+              // 
               currentChildNotis.vcnotis.forEach((item: any) => {
+                console.log(item,"..currentChildNotis");
+                if(item.title=="vcNoti1"){
+                const vcNotisExists= getVaccinesForPeriodCount(allVaccineData, item.growth_period);
```
Comment (Akhror) [inline app/components/HeaderNotiIcon.tsx:191]: Please fix the indentation, spacing and format issues all over the file

File: app/components/HeaderNotiIcon.tsx (hunk 1 above)
Comment (Akhror) [inline app/components/HeaderNotiIcon.tsx:193]: null and undefined represent the absence of a value, while an empty value typically refers to an empty string ('') or an empty array ([]) so it is recommended to check for null or undefined first before checking for an empty value.

File: app/components/HeaderNotiIcon.tsx (hunk 2 above)
Comment (Kalyani790) [reply app/components/HeaderNotiIcon.tsx]: Noted and done

**Pull request merged**